cinch/roles/jenkins_slave/templates/swarm.service

```diff
@@ -2,7 +2,20 @@
 After=network-online.target
 Wants=network-online.target
 [Service]
+User={{ jenkins_user }}
 EnvironmentFile=/etc/sysconfig/jenkins_swarm
+Restart=on-failure
+
+# Wait 10 seconds (swarm client default) for the service to fail; code in
```

Comment:
If code elsewhere depends on this, and we want it to match in these two locations, should we pull it out to a single variable so we don't rely on a magic value?